eZ/Bundle/EzPublishCoreBundle/Resources/views/default/content/embed.html.twig

```diff
@@ -0,0 +1,7 @@
+{% set content_name=ez_content_name(content) %}
+
+{% if location is defined %}
```

Comment:
When you render `ez_content:viewAction` controller with only `content` or `contentId` parameters.